background/update.js

```diff
@@ -15,11 +16,21 @@ var updater = {
   MAYBE_EDITED: 'may be locally edited',
   SAME_MD5: 'up-to-date: MD5 is unchanged',
   SAME_CODE: 'up-to-date: code sections are unchanged',
+  SAME_VERSION: 'up-to-date: version is unchanged',
   ERROR_MD5: 'error: MD5 is invalid',
   ERROR_JSON: 'error: JSON is invalid',
+  ERROR_VERSION: 'error: version is older than installed style',
 
   lastUpdateTime: parseInt(localStorage.lastUpdateTime) || Date.now(),
 
+  isSame(code) {
```

Comment:
I think it is a good idea to put error code definition and checker function in the same file?

Reply:
The name is too ambiguous (not a big problem) and currently `updater` has methods that deal with the update process in general (bigger problem) so I don't think it's a good idea to expose this function especially since it's used only once. Like I said, don't change the existing code in this PR unless absolutely necessary. You can refactor code in a separate PR later.